js/components/dataset/featured.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+<div class="btn-group btn-group-sm">
+    <button type="button"
+        class="btn btn-success featured"
+        :class="{active: featured}"
+        @click="toggleFeatured">
+        <span class="glyphicon glyphicon-bullhorn"></span>
```

Comment:
Might be better with font-awesome icons (ie. `fa fa-bullhorn`) so at some point we can drop bootstrap glyphicons totally and gain some Kb in the build